packages/components/popover/src/popperModifiers.ts

```diff
@@ -0,0 +1,18 @@
+import { Modifier } from '@popperjs/core';
+
+/**
+ * Sets the popover width to the size of the trigger element.
+ */
+export const sameWidth: Modifier<'sameWidth', any> = {
+  name: 'sameWidth',
+  enabled: true,
+  phase: 'beforeWrite',
+  requires: ['computeStyles'],
+  fn: ({ state }) => {
+    state.styles.popper.width = `${state.rects.reference.width}px`;
+  },
+  effect: ({ state }) => () => {
+    const reference = state.elements.reference as HTMLElement;
+    state.elements.popper.style.width = `${reference.offsetWidth}px`;
+  },
+};
```

Comment:
Isn't this only being used by `Popover.tsx`?

I suggest moving it to that file, if it ever gets shared between to components then we can abstract

Just for maintainability purposes I think
It's also a personal preference of not jumping from file to file so... I don't know hahahah
not a blocker, more like a suggestion